odxtools/multiplexer.py

```diff
@@ -149,16 +149,8 @@ def __repr__(self) -> str:
 
 def read_switch_key_from_odx(et_element):
     """Reads a Switch Key for a Multiplexer."""
-    byte_position = (
-        int(et_element.find("BYTE-POSITION").text)
-        if et_element.find("BYTE-POSITION") is not None
-        else None
-    )
-    bit_position = (
-        int(et_element.find("BIT-POSITION").text)
-        if et_element.find("BIT-POSITION") is not None
-        else 0
-    )
+    byte_position = int(et_element.findtext("BYTE-POSITION"))
```

Comment:
byte position is optional, int(None) will fail

Reply:
fixed